services/workers/src/test/java/org/sagebionetworks/schema/worker/JsonSchemaWorkerIntegrationTest.java

```diff
@@ -169,9 +170,12 @@ public void testCreateSchemaCycle() throws InterruptedException, AssertionError,
 		assertEquals("Schema $id: 'my.org.net-one' has a circular dependency", message);
 	}
 
-	public CreateSchemaResponse registerSchemaFromClasspath(String name) throws Exception {
```

Comment:
helpers:
1 getSchemaFromName()
2. registerSchema(schema)

Allows more generic schema modification between steps one and two.